load.php

```diff
@@ -260,3 +260,65 @@ function perflab_load_active_and_valid_modules() {
 if ( is_admin() ) {
 	require_once PERFLAB_PLUGIN_DIR_PATH . 'admin/load.php';
 }
+
+/**
+ * Trigger actions when a module gets activated or deactivated.
+ *
+ * @since n.e.x.t
+ *
+ * @param mixed $old_value Old value of the option.
+ * @param mixed $value     New value of the option.
+ */
+function perflab_run_module_activation_deactivation( $old_value, $value ) {
```

Comment:
It would be great if you could add a unit test for this function. To be fair, this function is part of the infrastructure, so it doesn't technically need to be in this PR, but we should definitely add a test for it one way or another to have this critical infrastructure piece covered.

Let me know if you prefer this to be done in a separate PR.

Reply:
This function was untestable with the PHPUnit test-modules because the test modules are not actually valid/loaded since they are not in the right paths in the plugin, so I added separate `perflab_activate_module` and `perflab_deactivate_module` functions, and added tests to those 👍